README.md

```diff
@@ -93,7 +93,7 @@ To enable the `tokyonight` colorscheme for `Lightline`:
 
 ```vim
 " Vim Script
-let g:lightline = {'colorscheme': 'tokyonight'}
+let g:lightline.colorscheme = 'tokyonight'
```

Comment:
The example is correct, both are a dictionary. If you are using this form, you'll need to set `let g:lightline = {}` first

Reply:
Ah, my bad. I did a straight copy and paste and it wasn't working. I am fine to close it.